### PR TITLE
fix(swagger): remove the generation of useless builtin types

### LINKED ISSUE #921: A String BodyParams
## Information

- **Version:** 5.59.1
- Packages: swagger

With the method with a simple string as BodyParams, swagger will create a Model for the string variable. And I mean it should not generate a Model

## Example

```typescript
@Controller('/')
class MyController {
    @ContentType('text/plain')
    @Post("/insert")
    async csv(@BodyParams() csvLines: string): Promise<string> {
       ....
    }
}
```

Output json:

```json
{
   "in": "body",
   "name": "body",
    "schema": {
        "type": "object",
        "properties":{}
    } 
}
```

Expected:

```json
{
   "in": "body",
   "name": "body",
    "schema": {
        "type": "string"
    } 
}
```

### PR DESCRIPTION
**Pull request by rluvaton**
## Informations

Type | Breaking change
---|---
Fix |No

****

## Description
Also, add tests to ensure these mistakes won't happen

Closes #921 

## Todos

- [x] Tests
- [x] Coverage
- [ ] Example
- [ ] Documentation

### CODE REVIEW

File: packages/swagger/src/class/OpenApiModelSchemaBuilder.ts [hunk 1]
```diff
@@ -32,6 +32,11 @@ export class OpenApiModelSchemaBuilder {
    * @returns {OpenApiModelSchemaBuilder}
    */
   build(): this {
+    // Stop from creating primitive schemas (#921)
+    if (isPrimitiveOrPrimitiveClass(this.target)) {
+      return this;
+    }
```
Comment (rluvaton) [inline packages/swagger/src/class/OpenApiModelSchemaBuilder.ts:38]: Is this the right thing to do? or it will mess with something?

Comment (Romakita) [reply packages/swagger/src/class/OpenApiModelSchemaBuilder.ts:38]: Yes is good for me ;)

**Review by Romakita — APPROVED** [review]

**Comment by Romakita** [comment]
Merged :) 

Thanks @rluvaton for your contribution ;)

**Pull request merged**